src/main/java/core/basesyntax/model/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.model;
+
+public class Square extends Color {
```

Comment:
Class names should not be verbs or actions. In this case, 'Square' is an appropriate name for a class representing a geometric figure.